docs/docs/config.md

```diff
@@ -26,19 +27,31 @@ password=PASSWORD             | mysql password
 host=HOST                     | mysql host
 port=PORT                     | mysql port
 producer=stdout,kafka,        | where shall we send these rows, sir?
-kafka.*=XXX                   | any options prefixed with 'kafka.' will be passed into the kafka producer library
+kafka.maxwell.topic           | provide a topic for maxwell to write to. Default will be "maxwell".
```

Comment:
maybe let's give kafka the `kafka.` namespace entirely, make this "kafa_topic" instead...